packages/consent/consent-tools/src/domain/create-wrapper.ts

```diff
@@ -180,3 +183,20 @@ const disableIntegrations = (
   )
   return results
 }
+
+const isDisabled = (
+  cdnSettings: CDNSettings,
+  initialCategories: Categories,
+  disableOpt: InitOptions['disable'] | undefined
```

Comment:
Should we have this shortcircuit before everything? As in, if `disable` is `true` then bypass all the cdnsettings function

Reply:
I actually was in the process of refactoring this when you wrote this comment =)